NLog.Targets.OpenTelemetryProtocol/OtlpTarget.cs

```diff
@@ -140,44 +140,73 @@ private ResourceBuilder CreateResourceBuilder()
 
         protected override void Write(LogEventInfo logEvent)
         {
-            var data = new LogRecordData
+            var formattedMessage = RenderLogEvent(Layout, logEvent);
+
+            var data = new LogRecordData()
             {
                 SeverityText = logEvent.Level.ToString(),
+                Severity = ResolveSeverity(logEvent.Level),
                 Timestamp = logEvent.TimeStamp,
-                Body = logEvent.FormattedMessage
+                Body = formattedMessage,
             };
 
-            var attributes = new LogRecordAttributeList { { OriginalFormatName, logEvent.Message } };
+            var attributes = new LogRecordAttributeList();
+            AppendAttributes(logEvent, ref attributes);
+            _logger.EmitLog(data, attributes);
+        }
 
-            if (logEvent.Parameters is not null && logEvent.Parameters.Length > 0)
-                AppendParameters(logEvent, ref attributes);
+        private void AppendAttributes(LogEventInfo logEvent, ref LogRecordAttributeList attributes)
+        {
+            if (logEvent.Exception != null)
+                attributes.RecordException(logEvent.Exception);
+
+            attributes.Add(OriginalFormatName, logEvent.Message);
 
-            if (_attributes.Count > 0)
+            if (IncludeEventParameters && logEvent.Parameters?.Length > 0)
```

Comment:
Please change this to 
`if (!ShouldIncludeProperties(logEvent) && IncludeEventParameters && logEvent.Parameters?.Length > 0)`
Otherwise, if you did something like `logger.Fatal("message: {param}", "testing")`, the parameter would be in the attributes twice

Reply:
Changed it only outputs `logEvent.Parameters` when not including LogEvent-properties.